langchain/retrievers/weaviate_hybrid_search.py

```diff
@@ -62,14 +62,14 @@ def add_documents(self, docs: List[Document]) -> List[str]:
         return ids
 
     def get_relevant_documents(
-        self, query: str, where_filter: Optional[Dict[str, object]] = None
+        self, query: str, where_filter: Optional[Dict[str, object]] = None, vector: Optional[List[float]] = None
```

Comment:
hm we shouldn't be changing this function signature since it's defined in BaseRetriever interface. in fact we probably shouldn't have added `where_filter` here either. imo we should define a separate function (eg `get_documents_with_filter`) that takes in both the filter and vector. cc @hwchase17